test/internal/testenv/testenv.go

```diff
@@ -40,6 +41,39 @@ func KongTag() string {
 	return os.Getenv("TEST_KONG_TAG")
 }
 
+// KongImageFullName returns the full name of Kong image.
```

Comment:
We might want to specify the env variable names used to get the return value.

Reply:
done.